internal/rbac/main.go

```diff
@@ -114,6 +115,22 @@ func HasRole(user types.User, role RoleName) (bool, error) {
 	return hasRole, types.NewErrServerError(err)
 }
 
+func GetUsersWithRole(role RoleName) ([]types.User, error) {
```

Comment:
I think this should be
```go
func GetUserIdsWithRole(role RoleName) ([]uuid.UUID, error) {
```
as, as a consumer I'd expect the `User`s returned would have the username set (and creation time etc.)

Reply:
the casbin method returns a slice of strings and the `UserById` service takes in a string, so can I just return strings or is it worth converting them to the uuid type? 🤔